server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/CassandraMailQueueView.java

```diff
@@ -68,7 +68,7 @@ public MailQueueView create(MailQueueName mailQueueName) {
 
     @Override
     public CompletableFuture<Void> storeMail(Instant enqueuedTime, Mail mail) {
-        return storeHelper.storeMailInEnqueueTable(mail, mailQueueName, enqueuedTime);
+        return storeHelper.storeMail(mail, mailQueueName, enqueuedTime);
```

Comment:
(not related to this line)

`CassandraMailQueueView.getSize()` should use `cassandraMailQueueBrowser.browseReferences()` method instead of the heavy `browse()`